USING-DEVNETS.md

```diff
@@ -0,0 +1,17 @@
+To use starknet devnet you have to export evironment variables first:
+
+```bash
+export $(cat development.env | xargs)
```

Comment:
`source development.env`

Reply:
Source creates shell variables instead of env variables as source expects the file to be a shell script

Reply:
Ok, there is however an option to do it with source and still preserve the .env syntax